### PR TITLE
Stop displaying the same artist/albums multiple times

### PR DESCRIPTION
**Pull request by jason-a69**
Changes made for issue #139

### CODE REVIEW

File: mopidy_gmusic/library.py [hunk 1]
```diff
@@ -311,14 +313,30 @@ def sorter(track):
     def refresh(self, uri=None):
         self.tracks = {}
         self.albums = {}
+        self.albums2 = {}
```
Comment (belak) [inline mopidy_gmusic/library.py]: Could this be named something a bit more descriptive? It's always nice to be able to look at a variable and get a bit of context about how it's used.

File: mopidy_gmusic/library.py [hunk 2]
```diff
@@ -311,14 +313,30 @@ def sorter(track):
...
 
             self.tracks[mopidy_track.uri] = mopidy_track
-            self.albums[mopidy_track.album.uri] = mopidy_track.album
+
+#           Need another albums list that will store all albumIds so they can be referenced in the browsing later
+#           Problem caused by Google assigning multiple albumIds for the same album
+
+            self.albums2[mopidy_track.album.uri] = mopidy_track.album
+            album_found = False
+            if len(self.albums) > 0:
```
Comment (belak) [inline mopidy_gmusic/library.py]: I would just drop this if-else and use the for loop all the time. Looping over an empty list shouldn't be very expensive.

File: mopidy_gmusic/library.py (hunk 1 above)
Comment (jason-a69) [reply mopidy_gmusic/library.py]: Can do, no problem

File: mopidy_gmusic/library.py (hunk 2 above)
Comment (jason-a69) [reply mopidy_gmusic/library.py]: OK, sure

**Comment by jason-a69** [comment]
I will put in a check for artist / album name to make sure album does not exist

**Comment by belak** [comment]
I'll try to take another look later. Please make sure flake8 (code style checks) passes for the portions you've changed (It's the part of the travis-ci build that's failing).

**Comment by kingosticks** [comment]
You know you can run flake8 locally, right? The whole Travis run can be reproduced locally with tox.

**Comment by jason-a69** [comment]
Hi Nick,

Thanks for this. I didn't know you could check locally, I should have investigated really.  Trying to find that misplaced space has been doing my head in.

Cheers

Jason

**Comment by kingosticks** [comment]
I have been there myself. https://docs.mopidy.com/en/latest/devenv/# is really worth doing. It doesn't take that long to setup, it just looks a lot because there's ample helpful explanation around the steps. 

You might also find `git commit --amend` and `git push --force` useful when you're updating a PR with small fixes.

**Comment by jjok** [comment]
This PR is no longer needed as the issue has been fixed by #233.

**Pull request closed without merging**